recipe/meta.yaml

```
@@ -18,42 +18,50 @@ build:
   entry_points:
     - jirashell = jira.jirashell:main
   number: 0
-  script: python -m pip install --no-deps --ignore-installed .
+  script: python -m pip install --no-deps --no-build-isolation --ignore-installed .
```

```suggestion
  script: python -m pip install --no-deps --no-build-isolation --ignore-installed . -v
```
`-v` will give us a little bit more information that is very useful when debugging.